src/components/BlogCard/BlogCard.jsx

```diff
@@ -2,24 +2,25 @@ import React from "react";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import "@components/BlogCard/blogCard.scss";
-import defaultUserImage from "@assets/images/icons/default.png";
 
 const BlogCard = ({ blog }) => {
-  const { id, title, tags, image, createdAt } = blog;
+  const { id, title, tags, image, createdAt, creatorFullName, creatorImage } =
+    blog;
+
   return (
     <div className="blog-card">
       <Link to={`/blog/${id}`} className="blog-card__link">
         <img src={image} className="blog-card__banner" alt="Blog Banner" />
         <div className="blog-card__category-badge">{tags}</div>
-        <h3 className="blog-card__blog-title">{title}</h3>
+        <h3 className="blog-card__title">{title}</h3>
         <div className="blog-card__author-info">
           <img
-            src={defaultUserImage}
+            src={creatorImage}
             alt="Author"
-            className="blog-card__author-info__author-image"
+            className="blog-card__author-image"
```

Comment:
```suggestion
            className="blog-card__author-info--author-image"
```
do like this where needed.
 `blog-card__author-info--author-image` here, `blog-card` is Block, `author-info` is Element and `author-image` is Modifier.

Reply:
Here ,in this card image is an element. So i take it as an element apu